src/FormModelInterface.php

```
@@ -145,72 +168,25 @@ public function getPlaceholders(): array;
      * Empty array is returned if no error.
      * If `$first` is `true`, only the first error is returned.
      *
-     * @psalm-return array<string>|string
+     * @phpstan-return array<string>|string
      */
     public function getPropertyError(string $property, bool $first = false): array|string;
```

PHPMD (2.15.0)

[error] The method getPropertyError has a boolean flag argument $first, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)